src/box/lua/config/instance_config.lua

```diff
@@ -119,32 +92,21 @@ end
 -- Available only in Tarantool Enterprise Edition.
 local function enterprise_edition(schema_node)
     schema_node.enterprise_edition = true
+    schema_node.validate = enterprise_edition_validate
 
     -- Perform a domain specific validation first and only then
     -- check, whether the option is to be used with Tarantool
     -- Enterprise Edition.
     --
     -- This order is consistent with data type validation, which
     -- is also performed before the EE check.
-    schema_node.validate = chain2(
-        schema_node.validate,
-        enterprise_edition_validate)
     schema_node.apply_default_if = chain2(
-        schema_node.apply_default_if,
+        schema_node.applygit_default_if,
```

Comment:
Typo?

Reply:
Oops, looks like some console input slipped in by accident.
Fixed it, thanks!